src/views/home.hbs

```diff
@@ -1,8 +1,7 @@
-
 <div id="search">
```

Comment:
i see those same lines of code in 2 files, it would make sense here for this to be a partial ( the search form )

Reply:
@Karyum separated it into a partial